core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -429,6 +433,11 @@ public ContainerisedDockerCompose(List<File> composeFiles, String identifier) {
         addEnv("DOCKER_HOST", "unix:///docker.sock");
         setStartupCheckStrategy(new IndefiniteWaitOneShotStartupCheckStrategy());
         setWorkingDirectory(containerPwd);
+        Path dockerConfig = Paths.get(System.getProperty("user.home"), ".docker");
```

Comment:
I checked my `~/.docker/` folder, it seems to be also used for things like Docker Machine and even some 3rd parties.
I would mount `.docker/config.json` instead of `.docker/` to avoid mounting something really big :)

Reply:
Could you please mention your change in `CHANGELOG.md` as well? :)